src/components/calculator.js

```diff
@@ -0,0 +1,54 @@
+import './calculator.css';
+
+const Input = () => <div className='input-wrapper'>0</div>;
+
+const Button = ({ symbol, color, double }) => (
+      <div
+         className="button-wrapper"
+        style={{ backgroundColor: color, width: double }}
+        >
+        {symbol}
+      </div>
+);
+
+const Calculator = () => (
+    <div className='calc-wrapper'>
+        <Input></Input>
+        <div className='raw'>
+        <Button symbol="A/c"/>
```

Comment:
- [ ] `Calculator.js` should have at least 1 child component. I suggest you to make the `Button` component external and import it in `Calculator.js`